### ISSUE TITLE
No Content-type found in the mock response

### ISSUE THREAD

**Issue by arunvelusamyd** [authorjapps/zerocode#212]
Mock response doesn’t have Content-type in the header , even if we configure it in the mock response.
```
"response": {
              "status": 200,
               “headers”: {
                    “Content-type” : “application/json”
               },
              "body": —response body—
}
```

**Comment by aghorpade** [comment]
@arunvelusamyd  - Content-type is part of Response Headers and not available as a separate entity. Checkout Readme file as below 
![image](https://user-images.githubusercontent.com/5318345/53468094-d7a95580-3a7e-11e9-8ecc-fa52d7bf5565.png)
test case response should as below -

"response": {
"status": 200,
"headers":{ 
      “Content-type” : “application/json”
  },
"body": —response body—
}

Let us know if you still face any issue

**Comment by authorjapps** [comment]
Seems like, @arunvelusamyd is fixing this in branch `ISS-212-mock-response-content-type` (work in progress). 

For time being, we can proceed with the PR and unblock Mr Arun. 👍 

Note:
#### 1)
This repo https://github.com/authorjapps/api-mock-maker supports all things around mocking and very flexible n easy to create mocks. Also uses JSON DSLs.
We should try to use this for our mocking if we can maintain the mocks separately.

#### 2)
If not, then we can integrate the above code into core module i.e. situations where we want the test-cases to have the mocks in the same JSON file(good for Integration Testing for a Dev-Build or commit-build in Jenkins) as earlier steps. The drawback in this approach is the test-cases become verbose for end-to-end testing as we don't need mocks for black-box testing.

Hence both have merits and pain, let's support both approaches going forward.
Let's do it in a separate ticket 👍 .

**Comment by aghorpade** [comment]
@authorjapps  - I don't think we need to fix anything here. Whenever any REST api respond then content comes either part of Header or Body section. Then why do we need to created new entity apart from header/body.
Not sure if I am missing any other information or didn't understand what you are fixing.

**Comment by arunvelusamyd** [comment]
@aghorpade 
Sorry, it was typo. It was part of response headers. Update my comment.

**Comment by authorjapps** [comment]
PR has been merged. This will be available in `1.3.4`

**Comment by authorjapps** [comment]
Released now.

```
<dependency>
  <groupId>org.jsmart</groupId>
  <artifactId>zerocode-tdd</artifactId>
  <version>1.3.4</version>
</dependency>
```

Maven Link: https://search.maven.org/artifact/org.jsmart/zerocode-tdd/1.3.4/jar
